README.md

```diff
@@ -109,8 +110,65 @@ MIT/Expat
 # Contributing
 
 See [CONTRIBUTING.md](CONTRIBUTING.md) if you are interested in internals or
-contributing to the project.
-
+contributing to the project. The original repository provided a [.zenodo.json](.zenodo.json)
+file, and we generate a [.contributors file](.all-contributorsrc) from that via:
```

Comment:
just now spotted that this is in README.md .  All of the content below is more pertinent to CONTRIBUTING.md, not user-facing README.md, please move